bitnami/clickhouse/README.md

```diff
@@ -32,6 +32,10 @@ Bitnami charts can be used with [Kubeapps](https://kubeapps.com/) for deployment
 - PV provisioner support in the underlying infrastructure
 - ReadWriteMany volumes for deployment scaling
 
+> **Tip**: If use Kubernetes 1.18, need commented follow code in values
```

Comment:
Thanks for your contribution. In order to follow a similar approach to other READMEs in the repository, I proposed some minor changes.
```suggestion
> If you are using Kubernetes 1.18, the following code needs to be commented out.
```